keyboards/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Please remove this file.

Reply:
done, I don't know if it was necessary but I removed the file AND the folder after comparing to other contributor's repository and noted no one had that foldet in /keyboards, probably I moved the folder by accident

Reply:
> Please remove this file.
This is strange, Travis-ci is failing for errors in other contributor's keyboards that are not mine, help

Reply:
That's fine.  Sometimes Travis CI ... doesn't know what it's doing.   As long as your keyboard compiles, that's what we're looking for.

Reply:
You may need to rebase this PR onto current master for Travis to see the correct changes and build accordingly.

Reply:
> You may need to rebase this PR onto current master for Travis to see the correct changes and build accordingly.

Please tell that doesn't mean I have to go through the entire PR process again :(

Reply:
Nope, you can just rebase and force push to the same branch and the PR will be updated.

Reply:
I don't understand what you mean by "rebase"

Reply:
This: https://docs.qmk.fm/#/newbs_git_resolving_merge_conflicts?id=rebasing-your-changes

Reply:
That fixed it! thanks! :)